src/main/java/io/r2dbc/postgresql/message/backend/AbstractCopyResponse.java

```diff
@@ -88,11 +88,6 @@ public String toString() {
         switch (count) {
             case 0:
                 return EnumSet.noneOf(Format.class);
-            case 1:
```

Comment:
This could be even more simplified by removing `switch` entirely. When `count` is zero, then we don't even enter the loop.

Reply:
Sure, I agree it's better.